polynote-kernel/src/main/scala/polynote/kernel/remote/transport.scala

```diff
@@ -292,10 +306,11 @@ object SocketTransport {
           str => if (str contains " ") s""""$str"""" else str
         }.mkString(" ")
 
-        val processBuilder = new ProcessBuilder(command: _*).inheritIO()
+        val processBuilder = new ProcessBuilder(command: _*).redirectErrorStream(true)
```

Comment:
Now the subprocess stdout and stderr is being pushed into `Logging` via `logProcess` up there. There's probably a better way to do it...